pkgs/applications/networking/instant-messengers/slack/default.nix

```diff
@@ -73,7 +73,8 @@ in stdenv.mkDerivation {
 
     # Fix the symlink
     rm $out/bin/slack
-    ln -s $out/lib/slack/slack $out/bin/slack
+    makeWrapper $out/lib/slack/slack $out/bin/slack \
+      --set LD_LIBRARY_PATH ${rpath}
```

Comment:
The slack binary itself doesn't have libxkbfile.so as a dependency. Wouldn't the path to libkbfile be removed from the rpath in that case?